Makefile

```diff
@@ -133,5 +137,26 @@ docu_check:
 docu_clean:
 	make -C documentation clean
 
+# Targets related to kubebuilder
```

Comment:
On my machine, this target will use any controller-gen you have on the path if it exists. If you don't have controller-gen on the path, it will download it and use it, which I think is the behavior you want?

Reply:
Yes, but if it doesn't exists. Wouldn't it execute `go get` every time again? My idea was to leverage the functionality of make, to create a local version of the binary (using `go get`) and then use it.